docker-compose.yml

```diff
@@ -48,3 +48,6 @@ services:
         image: mariadb:10.2.15
         env_file: .env
         restart: unless-stopped
+        command: ["mysqld", "--default-authentication-plugin=mysql_native_password"]
+        ports:
+            - 3306:3306
```

Comment:
We don't want to expose our database! The credentials are open and database should be accessible only to the isolated network of containers

Reply:
Oops, didn't mean to commit that, was just exposing it locally for debugging. 😛

Reply:
Okay!